src/DonorProfiles/Profile.php

```diff
@@ -40,9 +42,11 @@ public function update( $data ) {
 
 		$pipeline = ( new DonorProfilePipeline )
 			->pipe( new UpdateDonorName )
+			->pipe( new UpdateDonorCompany )
 			->pipe( new UpdateDonorAvatar )
 			->pipe( new UpdateDonorEmails )
-			->pipe( new UpdateDonorAddresses );
+			->pipe( new UpdateDonorAddresses )
+			->pipe( new UpdateDonorAnonymousGiving );
```

Comment:
Pipe-able classes should implement an interface - sorry for not catching this sooner.

Reply:
Done, ref: https://github.com/impress-org/givewp/pull/5633/commits/3402bfe77a478ff8084cc196a11fa307313e1854